packages/cli/src/package.json.spec.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+import { flatten } from 'lodash';
+import { listMonorepoPackageNames } from 'common';
+import * as packageJson from '../package.json';
+
+describe('package.json dependencies', () => {
```

Comment:
How this test will fail if CLI package has missing external dependencies? The ideal output would be diff of external dependencies that are missing from the current CLI package.json

Reply:
Updated the test to consolidate all missing deps into one output. An example of the new failure text (from removing @medv/finder and raw-body temporarily from package.json):

```
  ● package.json dependencies › includes direct dependencies for each non-monorepo dependency of its transitive monorepo dependencies

    expect(received).toBe(expected) // Object.is equality

    - Expected  - 1
    + Received  + 4

    - Array []
    + Array [
    +   "package.json needs a dependency on \"@medv/finder\": \"^2.0.0\" (required via accessibility-insights-crawler)",
    +   "package.json needs a dependency on \"raw-body\": \"^2.4.1\" (required via accessibility-insights-crawler, axe-result-converter, common, scanner-global-library)",
    + ]

      47 |             });
      48 |
    > 49 |         expect(missingDependencyExplanations).toBe([]);
```